challenge-6/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  </head>
+    <input type="text" id="number"></input>
+    <button id="test">Test</button>
+  <body>
+    <script>
+      $('#test').click(() => {
+        let response = $('#number').val();
+        response = response.split('').map(Number);
+        if (response.length === 16 || response.length === 15){
+          for (var i = response.length - 2; i > -1; i = i - 2){
+            response[i] = response[i] * 2;
+            if (response[i] > 9) {
+              response[i] = response[i] - 9;
+            }
+          }
+          let add = 0;
+          for (var x = 0; x < response.length; x++) {
```

Comment:
There's actually a cool function similar to `map` that does just this! It's called [`.reduce()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce).